semantic-ast/BUILD.bazel

```diff
@@ -46,3 +47,37 @@ haskell_library(
         "@stackage//:unordered-containers",
     ],
 )
+
+all_ts_deps = ["@tree-sitter-{name}//:tree-sitter-{name}".format(name = name) for name in [
+    "go",
+    "java",
+    "json",
+    "php",
+    "python",
+    "ql",
+    "ruby",
+    "rust",
+    "tsx",
+    "typescript",
+]]
+
+haskell_binary(
+    name = "generate-ast",
+    srcs = glob(["app/**/*.hs"]),
+    compiler_flags = GHC_FLAGS + EXECUTABLE_FLAGS + ["-XStrictData"],
+    deps = [
+        ":semantic-ast",
+        "//semantic-source",
+        "//:base",
+        "//:filepath",
+        "//:process",
+        "//:template-haskell",
+        "//:text",
+        "@stackage//:directory",
+        "@stackage//:generic-lens",
+        "@stackage//:lens",
+        "@stackage//:tree-sitter",
+        "@stackage//:neat-interpolation",
+        "@stackage//:optparse-generic",
+    ] + all_ts_deps,
+)
```

Comment:
I haven’t added this to `semantic-ast.cabal`, reasoning that we’re not going to run this anywhere but our local machines, that expressing it in the cabal format would be tedious with few upsides, and that I want people to start migrating to Bazel!